python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -764,8 +764,10 @@ def do_evaluate(
             for c, dtype in zip(df.columns, schema.values(), strict=True)
         )
         if predicate is not None:
```

Comment:
I _think_ `predicate` is always `None.`

Reply:
@wence- does the IR never produce pushdown instructions for us? I know Polars supports it in some of their readers.